### PR TITLE
vim-patch:partial:f1dcd14fc5d4

### PR DESCRIPTION
**Pull request by clason**
Update runtime files

https://github.com/vim/vim/commit/f1dcd14fc5d4370476cd82895a4479ca2d252e54

Co-authored-by: Bram Moolenaar <Bram@vim.org>

### CODE REVIEW

**Review by seandewar — COMMENTED** [review]
The vimNotFunc change in `syntax/vim.vim` looks relevant.

However, the vimCollClass and vimNotation changes look like errors to me?
Maybe skip those.

File: runtime/doc/diff.txt [hunk 1]
```diff
@@ -137,6 +137,10 @@ Otherwise they are set to their default value:
 	'foldmethod'	"manual"
 	'foldcolumn'	0
 
+'foldenable' will most-likely be reset to off.  That is when 'foldmethod' is
+is restored to "manual".  The folds themselves are not cleared but they should
```
Comment (smjonas) [inline runtime/doc/diff.txt:141]:
```suggestion
restored to "manual".  The folds themselves are not cleared but they should
```

**Comment by clason** [comment]
> The vimNotFunc change in `syntax/vim.vim` looks relevant.
> 
> However, the vimCollClass and vimNotation changes look like errors to me? Maybe skip those.

Isn't that the same change every time?

**Comment by seandewar** [comment]
> Isn't that the same change every time? 

Yes, but vimNotation is used for keycodes and vimCollClass is used for collection classes in patterns. `<retu>`/`<retur>` and `[[:retu:]] `/`[[:retur:]] ` don't exist.

The change makes sense for vimNotFunc though, as that concerns highlighting `:retu`/`:retur`, which are valid short forms of `:return`.
Presumably it was added so that a `:retu`/`:retur` used like:
```vim
func F()
    retu(1337)
endfunc
```
is not highlighted as a function call. 

I guess somebody was a bit too overzealous with `:substitute`. :P

File: runtime/syntax/vim.vim [hunk 2]
```diff
@@ -423,7 +423,7 @@ syn match	vimMenuBang	"!"	contained skipwhite nextgroup=@vimMenuList
 " Angle-Bracket Notation: (tnx to Michael Geddes) {{{2
 " ======================
 syn case ignore
-syn match	vimNotation	"\%#=1\(\\\|<lt>\)\=<\([scamd]-\)\{0,4}x\=\(f\d\{1,2}\|[^ \t:]\|cmd\|cr\|lf\|linefeed\|return\|enter\|k\=del\%[ete]\|bs\|backspace\|tab\|esc\|right\|left\|help\|undo\|insert\|ins\|mouse\|k\=home\|k\=end\|kplus\|kminus\|kdivide\|kmultiply\|kenter\|kpoint\|space\|k\=\(page\)\=\(\|down\|up\|k\d\>\)\)>" contains=vimBracket
+syn match	vimNotation	"\%#=1\(\\\|<lt>\)\=<\([scamd]-\)\{0,4}x\=\(f\d\{1,2}\|[^ \t:]\|cmd\|cr\|lf\|linefeed\|retu\%[rn]\|k\=del\%[ete]\|bs\|backspace\|tab\|esc\|right\|left\|help\|undo\|insert\|ins\|mouse\|k\=home\|k\=end\|kplus\|kminus\|kdivide\|kmultiply\|kenter\|kpoint\|space\|k\=\(page\)\=\(\|down\|up\|k\d\>\)\)>" contains=vimBracket
```
Comment (seandewar) [inline runtime/syntax/vim.vim:426]: Make this change for vimNotFunc instead

Comment (clason) [reply runtime/syntax/vim.vim:426]: 🤦

**Review by seandewar — APPROVED** [review]
LGTM.
Maybe partial due to missing changes for unported stuff like autocmd blocks and getscriptinfo() (and maybe E1336)? They're small, though.

**Review by zeertzjq — APPROVED** [review]

**Pull request merged**